tfjs-backend-wasm/cloudbuild.yml

```diff
@@ -56,12 +56,13 @@ steps:
   waitFor: ['build']
 
 # Check bundle size.
-- name: 'gcr.io/learnjs-174218/wasm'
-  dir: 'tfjs-backend-wasm'
-  id: 'test-bundle-size'
-  entrypoint: 'yarn'
-  args: ['test-bundle-size']
-  waitFor: ['yarn', 'build-deps', 'build']
+# TODO(annxingyuan): Reenable once https://github.com/tensorflow/tfjs/pull/4012 is merged.
```

Comment:
Just curious why this refers to this PR itself (given that the fix is in the PR)? I'm sure there is a reason, just curious to what it might be.

Reply:
This reminds me I need to move bundle testing to nightly asap! ^_^